.github/workflows/scorecards.yml

```diff
@@ -51,6 +51,7 @@ jobs:
 
       # Upload the results to GitHub's code scanning dashboard.
       - name: "Upload to code-scanning"
-        uses: github/codeql-action/upload-sarif@5f532563584d71fdef14ee64d17bafb34f751ce5 # v1.0.26
+        uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: results.sarif
+          category: ossf-scorecard
```

Comment:
Is there a reason to not add newlines? I saw this in another patch.

Reply:
Unintentional